blocks/header/header.css

```
@@ -279,6 +279,13 @@ header nav#nav .nav-tools {
   grid-area: tools;
 }
 
+header nav#nav .nav-tools .button {
+  background: none;
+  border: none;
+  padding: 0;
```

I would set the padding, so that it is easier to click
```suggestion
  padding: 10px;
```

---

Done in https://github.com/adobe/helix-block-collection/pull/37/commits/70a0d5fa0fdf86c06623297ef2542026eb4cb673 👍